jenkins-master/docker-compose.test.yml

```diff
@@ -0,0 +1,5 @@
+version: "3"
+services:
+  sut:
+    build: .
```

Comment:
Can we be sure that it is build once?

Reply:
The documentation is not clear on that. From the logs, it looks like the image is built twice but the second build (for the tests) simply reuses all layers from the first one.